src/plone/app/workflowmanager/browser/layout.py

```diff
@@ -0,0 +1,55 @@
+from Products.Five.browser import BrowserView
```

Comment:
why not using the conventions on imports on this new module? also, you're missing the magic encoding comment